incubator/hnc/pkg/reconcilers/hnc_config.go

```diff
@@ -218,12 +244,28 @@ func (r *ConfigReconciler) forceInitialReconcile(log logr.Logger, reason string)
 	}()
 }
 
+// getMode returns Mode field from a TypeSynchronizationSpec object. Please
+// see the comments of api.SynchronizationMode for currently supported modes.
+// If Mode is not set, it will be api.Propagate by default. Any unknown mode is
+// treated as api.Ignore.
+func (r *ConfigReconciler) getMode(t api.TypeSynchronizationSpec) api.SynchronizationMode {
+	switch mode := t.Mode; mode {
```

Comment:
Why not just `switch t.Mode`? I don't know this part of golang that well, is there a difference?

Reply:
You are right. We can just use `switch t.Mode`.